example-apps/internal-knowledge-search/app-ui/src/components/SearchApplicationSettings.tsx

```diff
@@ -51,17 +250,34 @@ const SearchApplicationSettings: React.FC = () => {
 
             <div className="text-left mb-6 p-4 border rounded bg-gray-50">
                 <label
-                    htmlFor="apiKey"
+                    htmlFor="appUser"
                     className="block text-sm font-medium mb-1 text-gray-700"
                 >
-                    API Key:
+                    Application Elasticsearch Username:
```

Comment:
Because it's the username for the user of the Search Application. Which is an Elasticsearch thing, so "Elasticsearch username" would technically be correct, but I wanted to make clear that this is the user executing a search against the Search Application (in the event that another search persona is not selected). WDYT?

Reply:
`Search Application User` might be clearer then? It's not a big deal anyway and I'm not sure my intuitions are correct here, I just stumbled over it because it seems very awkwardly phrased.